tests/test_flask_pyoidc.py

```diff
@@ -22,7 +24,7 @@
 from .util import signed_id_token
 
 
-class TestOIDCAuthentication(object):
+class TestOIDCAuthentication(unittest.TestCase):
```

Comment:
Why was this added?
~The test setup always uses `pytest` and no features from `unittest` so this should not be necessary.~
I saw it further down, with the assertions, but those should be changed to match the current style. 🙂